optimum/exporters/onnx/convert.py

```diff
@@ -154,17 +158,22 @@ def validate_model_outputs(
     if atol is None:
         atol = config.ATOL_FOR_VALIDATION
 
-    framework = (
-        "pt" if is_torch_available() and issubclass(type(reference_model), (PreTrainedModel, ModelMixin)) else "tf"
-    )
+    classinfo = (PreTrainedModel,)
+    if is_diffusers_available():
+        classinfo += (ModelMixin,)
+
+    if "diffusers" in str(reference_model.__class__) and not is_diffusers_available():
+        raise ImportError("The pip package `diffusers` is required to validate stable diffusion ONNX models.")
```

Comment:
Maybe make it a function if this is going to be repeated a few times in the functions of this file.

Reply:
I use it only here in the end

Reply:
It is used only twice here, I'd say it's OK